index.html

```diff
@@ -849,10 +849,65 @@
   This section has not been edited for narrative.
 </aside>
 
-We define <dfn data-lt="data">personal data</dfn> as any information that is directly or
-indirectly related to an identified or identifiable [=person=], such as by reference to an
-[=identifier=] ([[GDPR]], [[OECD-Guidelines]], [[Convention-108]]).
+Personal data is data describing a person, relating to that persons interests,
+actions or behaviors.
+
+What personal data is appropriate to collect or
+process is highly context dependent; in some cases a person will want to share
+some personal data with a website, whether that data is extremely specific
+to themselves (e.g., a person describing their interests and goals
+to create a dating profile), or very general and shared with many others
+(e.g., a person sharing what Web browser they use, so that a website can
+be tailored accordingly).
+
+Web browser vendors, specification writers, and site authors should only
+collect and process personal data when it helps the person that data describes
+achieve that persons goals.
+
+
+### Implied "Goals" For Private Data Must Be Immediate
+
+Parties should only collect personal data when it directly
+and unambiguously furthers that person's goals. It is
+not appropriate for a data-collecting site to infer those goals in broad
+and abstract ways (e.g., its not appropriate for a site author to reason
+"if the user visits my site, then it must be their goal to support the site;
+therefor the user wants me to collect their personal data for monetization
+purposes.").
+
+
+### Data Can "Personal Data" Even If It Doesn't Enable Identification
+
+Unwanted collection of "personal data" is harmful to people, even when
```

Comment:
This is related to data about others (PR#130).  how should this be tied to PR130?